phpcs.xml.dist

```diff
@@ -19,5 +19,22 @@
     <exclude-pattern>*/TestAsset/*</exclude-pattern>
 
     <!-- Include all rules from Laminas Coding Standard -->
-    <rule ref="LaminasCodingStandard"/>
+    <rule ref="LaminasCodingStandard">
+        <!--
+        declare(strict_types=1) is too risky for a security-only component,
+        and should only be reconsidered once this package receives new feature work.
+        -->
+        <exclude name="SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing"/>
+
+        <!--
+            Function libxml_disable_entity_loader() has been deprecated
+        -->
+        <exclude name="Generic.PHP.DeprecatedFunctions.Deprecate"/>
+        <exclude name="Generic.PHP.DeprecatedFunctions.Deprecated"/>
```

Comment:
If the `libxml_disable_entity_loader` function call cannot be removed, then disable the coding standard check there, but don't exclude it for everything.
Thanks in advance! 👍🏻

Reply:
I wouldn't worry about it until `psalm.xml` starts reporting it :+1:

Reply:
Which Psalm?

Reply:
Oh, poop, indeed! No psalm here @_@ 

I'll amend this, thanks for noticing!